cmd/cluster/aws/create.go

```diff
@@ -504,3 +499,46 @@ func validateAWSOptions(ctx context.Context, opts *core.CreateOptions, awsOpts *
 
 	return nil
 }
+
+func validateMultiArch(ctx context.Context, o *RawCreateOptions, opts *core.CreateOptions) error {
+	var validMultiArchImage bool
+
+	// Check if the release image is multi-arch
+	if len(opts.ReleaseImage) > 0 && len(opts.PullSecretFile) > 0 {
+		pullSecret, err := os.ReadFile(opts.PullSecretFile)
+		if err != nil {
+			return fmt.Errorf("failed to read pull secret file: %w", err)
+		}
+
+		validMultiArchImage, err = registryclient.IsMultiArchManifestList(ctx, opts.ReleaseImage, pullSecret)
+		if err != nil {
+			return err
+		}
+	}
+
+	// Check if a release stream was provided instead and its multi-arch
+	if len(opts.ReleaseStream) > 0 && !strings.Contains(opts.ReleaseStream, "multi") {
```

Comment:
```suggestion
	if len(opts.ReleaseStream) > 0 && strings.Contains(opts.ReleaseStream, "multi") {
```
shouldn't this be a positive check or did I misunderstand it?

Reply:
Yes 🤦🏻